src/main/java/it/gov/pagopa/idpay/transactions/service/commands/ops/DeleteInitiativeServiceImpl.java

```diff
@@ -20,27 +25,37 @@ public DeleteInitiativeServiceImpl(RewardTransactionRepository rewardTransaction
     }
 
     @Override
-    public Mono<String> execute(String initiativeId) {
-        log.info("[DELETE_INITIATIVE] Starting handle delete initiative {}", initiativeId);
-        return deleteTransactions(initiativeId)
-                .then(Mono.just(initiativeId));
+    public Mono<String> execute(QueueCommandOperationDTO payload) {
+        log.info("[DELETE_INITIATIVE] Starting handle delete initiative {}", payload.getEntityId());
+        return deleteTransactions(payload.getEntityId(),
+                Integer.parseInt(payload.getAdditionalParams().get(PAGINATION_KEY)),
+                Long.parseLong(payload.getAdditionalParams().get(DELAY_KEY)))
```

Comment:
Why are you reading these info from the EVENT?

The event cannot define the batchSize behavior for all the collections!
Each application should define these through application.yml, in which they could be overwritten by ENV